### ISSUE TITLE
Add possibility to process inspect json file content from stdin

### ISSUE THREAD

**Issue by munkie** [nexdrew/rekcod#13]
I had some troubles re running container that stopped on production
And i did have permissions to install you lib on prod server
Though i could get content of `docker inspect` command for stopped container
My proposal to be able to process this file locally
For emaple by `rekcod < inspect.json` command

**Comment by nexdrew** [comment]
@munkie Thanks for reporting!

This is similar to #7, but, if I understand you correctly, you want to be able to pipe the _JSON output_ directly to `rekcod` instead of just a container name or id? I think I can implement it to support both.

It'll take a bit of refactoring, but I'll try to add this within the next week. Thanks again!

**Comment by munkie** [comment]
Andrew, correct. I want to pipe json directly.

**Comment by munkie** [comment]
@nexdrew By the way thanks for this tool. It helped me to restart docker container on production, while our deployment system was down :+1:

**Comment by nexdrew** [comment]
@munkie Finally made time to work on this, see PR #14. I published the branch to the `next` tag, feel free to give this a shot if you get a chance:

```console
npm i -g rekcod@next
```

Then you can pipe JSON content, file names, or containers to rekcod:

```console
$ cat docker-inspect.json | rekcod
$ rekcod docker-inspect.json
$ docker ps -aq | rekcod
```

Once I add more tests and update the docs, I'll publish 2.0.0 to `latest`.

Let me know what you think, and thanks for your valuable feedback!

**Comment by nexdrew** [comment]
@munkie This functionality has been released with version 2.0.0.

```console
npm i -g rekcod@latest
```

This will allow you to save/copy the output of `docker inspect` remotely and then feed it to `rekcod` locally, so you don't need to install `rekcod` on your production servers.

Let me know what you think, and thanks for reporting!

**Comment by munkie** [comment]
Great work @nexdrew Thanks!
